File: week-2/lectures/scores.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int num = 3;
    int scores[num];

    for (int i = 0; i < num; i++)
    {
        scores[i] = get_int("Score: ");
    }

    int avg;
    for (int i = 0; i < num; i++)
    {
        avg += scores[i];
    }

    printf("Average: %f\n", (scores[0] + scores[1] + scores[2]) / 3.0);
}

File: week-6/lectures/rvw-2/scores.py
#!/usr/bin/python3

import cs50
import sys


def main():
    scores = []
    for i in range(3):
        score = get_score()
        scores.append(score)
    avg = sum(scores) / len(scores)
    print(f"Avg: {avg}")
    sys.exit(0)


def get_score():
    while True:
        score = cs50.get_int("Score: ")
        if score > 0:
            break
    return score


main()
